stix_shifter_modules/cbcloud/tests/stix_translation/test_cbcloud_json_to_stix.py

```diff
@@ -0,0 +1 @@
+# TODO
```

Comment:
you can take a look at other modules how the tests are written. there are not much difference while writing unit tests for translations